### PR TITLE
add inception-resnet-v2 in branch inception-resnet-v2

### PR DESCRIPTION
**Pull request by burness**
Hi @aymericdamien , As mentioned in #421, the inception-resnet-v2 not to be convergent in 17flower. Now I found the reason. I forget the `batch_norm` after each conv_2d in the model.  Now I fixed it and it run in my machine and everything is ok and add normalizer_fn and normalizer_params in the conv_2d because the batch_normalization must before the bias in the conv_2d. So I fixed it  in the conv_2d instread of in the example after the conv_2d. 
I am sorry that i forget the last pr in conv_3d_transpose and i have commit the inception-resnet-v2 in the pr #446 
Pls let me know how to revert the last commit in #446 

Thanks again for the TFlearn

### CODE REVIEW

**Comment by burness** [comment]
@aymericdamien , I close the PR #446 and make a pr again. Pls check the conv_3d_transpose in pr #451

**Comment by aymericdamien** [comment]
Thanks for adding inception-resnet-v2!
I am thinking if it is better to add conv2d parameter 'normalize_fn' or simply call batch_normalization like that:
```python
tower_conv1_0 = batch_normalization(conv_2d(net, 32, 1, activation='relu',bias=False, name='Conv2d_0a_1x1'))
```

**Comment by burness** [comment]
@aymericdamien Maybe  it is not right. In [Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift](https://arxiv.org/abs/1502.03167), it seems that the batch normalization should be before the activation.

**Comment by aymericdamien** [comment]
Right! I meant something like:
```
tower_conv1_0 = relu(batch_normalization(conv_2d(net, 32, 1, bias=False, name='Conv2d_0a_1x1')))
```
My point is removing "normalize" argument from conv_2d, because it already have many arguments and I think we should avoid to surcharge it too much.

**Comment by burness** [comment]
@aymericdamien  OK!  I will fix

**Comment by burness** [comment]
Hi, @aymericdamien  i have fix and test in 17flowers local. Pls review

**Comment by aymericdamien** [comment]
This looks great :) Thanks for adding that!

**Comment by ibadami** [comment]
I know this is not a right platform to ask questions not related to the bugs. But I cannot find the answer in the documentation hence posting it here.
Could you please tell me how to read the pre-trained weights for inception-resnet-v2 model from the checkpoint binary provided by tensor-flow on this link : https://github.com/tensorflow/models/tree/master/slim. 
I wish to fine tune this already trained model for a different dataset. I am using tflearn for its leas verbose frame work. I see that there is a very nice implementation for this model already provided for TFlearn community, however I cannot seem to find a way to load the weights here. Any help would be appreciated.

**Comment by ibadami** [comment]
@burness  After running your code inception_resnet_v2.py. I run tensorboard to see the graph of this model. The graph looks broken, it does not show proper input/output edges. When i removed the batch normalization, it shows the graph correctly. Alternatively the googlenet.py example also shows correct graph since it does not have batch normalization. Is this behavior normal?
